### PR TITLE
Fixes #31413 - GraphQL api

### PR DESCRIPTION
**Pull request by kamils-iRonin**

### CODE REVIEW

**Comment by lzap** [comment]
Looks good, feel free to finish this. Thank you.

**Comment by kamils-iRonin** [comment]
I decided to drop _Draft_ label, we can open a new PRs to add create/update/delete mutations

File: test/factories/webhook_target.rb [hunk 1]
```diff
@@ -1,9 +0,0 @@
-# frozen_string_literal: true
-
-FactoryBot.define do
-  factory :webhook_target do
-    sequence(:name) { |n| "Webhook Target #{n}" }
-    target_url { 'https://hook.example.com/api/callback' }
-    events { ['subnet_created.event.foreman'] }
-  end
-end
```
Comment (lzap) [inline test/factories/webhook_target.rb:9]: I did not even remember what was this used for, any idea @ofedoren ? :)

**Comment by ofedoren** [comment]
Since Ondrej Prazak is no longer here to review GraphQL related PRs, I need more time to get more familiar with the stuff and how we do that. Everything is looks excellent, but I don't like merge things I don't quite understand :/

**Comment by adamruzicka** [comment]
@kamils-iRonin Hi, I'm sorry this is taking so long to get in. Is it still up to date?

**Comment by kamils-iRonin** [comment]
> Hi, I'm sorry this is taking so long to get in. Is it still up to date?

@adamruzicka I'll rebase and we will see if the pipeline is green  :)

**Comment by adamruzicka** [comment]
Looks ok apart from the duplicate column definition

**Review by ofedoren — APPROVED** [review]
Thanks, @kamils-iRonin and @adamruzicka. LGTM, packit failure is expected (we just need to update RPM spec)

**Review by adamruzicka — APPROVED** [review]
All good

**Comment by adamruzicka** [comment]
Thank you @kamils-iRonin, both for your patch and your patience :)

**Pull request merged**